jobs/JGLOBAL_ATMOS_VMINMON

```diff
@@ -17,16 +17,16 @@ export gcyc=${GDATE:8:2}
 #############################################
 # TANKverf - WHERE OUTPUT DATA WILL RESIDE
 #############################################
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_ANALYSIS
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_MINMON
-YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx COM_ATMOS_MINMON_PREV:COM_ATMOS_MINMON_TMPL
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COMIN_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COMOUT_ATMOS_MINMON:COM_ATMOS_MINMON_TMPL
+YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx COMOUT_ATMOS_MINMON_PREV:COM_ATMOS_MINMON_TMPL
 
-export gsistat="${COM_ATMOS_ANALYSIS}/${RUN}.t${cyc}z.gsistat"
-export M_TANKverf=${M_TANKverf:-${COM_ATMOS_MINMON}}
-export M_TANKverfM1=${M_TANKverfM1:-${COM_ATMOS_MINMON_PREV}}
+export gsistat="${COMIN_ATMOS_ANALYSIS}/${RUN}.t${cyc}z.gsistat"
+export M_TANKverf=${M_TANKverf:-${COMOUT_ATMOS_MINMON}}
+export M_TANKverfM1=${M_TANKverfM1:-${COMOUT_ATMOS_MINMON_PREV}}
 
-if [[ ! -d ${M_TANKverf} ]]; then mkdir -p -m 775 "${M_TANKverf}" ; fi
-if [[ ! -d ${M_TANKverfM1} ]]; then mkdir -p -m 775 "${M_TANKverfM1}" ; fi
+if [[ ! -d ${M_TANKverf} ]]; then mkdir -p "${M_TANKverf}" ; fi
+if [[ ! -d ${M_TANKverfM1} ]]; then mkdir -p "${M_TANKverfM1}" ; fi
```

Comment:
Since this is only an input, we shouldn't be creating it.
```suggestion
```

Reply:
@WalterKolczynski-NOAA
From scripts ( exglobal_atmos_vminmon.sh), these three directory variables should be input directories. Hence line 28 and 29 should be removed. Am I right?

Reply:
Only `$M_TANKverfM1` is an input. `$M_TANKverif` is an output (the perl scripts write to it).

Reply:
@WalterKolczynski-NOAA 
After reading 3 perl files in ush (minmon_xtrct_costs.pl  minmon_xtrct_gnorms.pl  minmon_xtrct_reduct.pl),    gsistat should be only an input file defined in jobs/JGLOBAL_ATMOS_VMINMON. This implies the following modifications:
 20 YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
 21         COMIN_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL \
 22         COMOUT_ATMOS_MINMON:COM_ATMOS_MINMON_TMPL
 23
 24 YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
 25         COMOUT_ATMOS_MINMON_PREV:COM_ATMOS_MINMON_TMPL
 26
 27 export gsistat="${COMIN_ATMOS_ANALYSIS}/${RUN}.t${cyc}z.gsistat"
 28 export M_TANKverf=${M_TANKverf:-${COMOUT_ATMOS_MINMON}}
 29 export M_TANKverfM1=${M_TANKverfM1:-${COMOUT_ATMOS_MINMON_PREV}}
 30
 31 if [[ ! -d ${M_TANKverf} ]]; then mkdir -p "${M_TANKverf}" ; fi
 32 if [[ ! -d ${M_TANKverfM1} ]]; then mkdir -p "${M_TANKverfM1}" ; fi
 33

Reply:
What modifications are you proposing? That looks like what is there now.